pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -332,3 +332,12 @@ def predict_step(self, *args, **kwargs):
     def post_training_step(self):
         if not self.lightning_module.automatic_optimization:
             self.model.require_backward_grad_sync = True
+
+    @classmethod
+    def register_plugins(cls, plugin_registry: Dict) -> None:
+        plugin_registry.register(
+            "ddp_with_find_unused_parameters_false",
+            cls,
+            description="DDP Plugin with `find_unused_parameters` as False",
+            find_unused_parameters=False
+        )
```

Comment:
i'd prefer splitting this out to a separate PR so this PR fixes only the typing. wdyt?

Reply:
Actually, I found an issue with the Registry. Will do a separate PR for the fix and this.